src/utils/version.ts

```diff
@@ -49,3 +50,26 @@ export function extractVersionFromPackageLock(
     return undefined;
   }
 }
+
+export function extractVersionFromPackageJson(
+  packageFileContent: string,
+  packageName: string
+): string | undefined {
+  const packageJson = JSON.parse(packageFileContent);
+  const dependencies = packageJson.dependencies;
+  return dependencies[packageName];
+}
+
+export async function fetchNPMVersion(
+  packageName: string
+): Promise<string | undefined> {
+  try {
+    const response = await fetch(`https://registry.npmjs.org/${packageName}`);
```

Comment:
should we use github api rather than using npm?
since we use the tags and sync with the appropriate package managers, that way we can reuse for all 3 ios, react-native & flutter sdk?

the api url format is the following:
```
https://api.github.com/repos/OWNER/REPO/releases/latest
```

Reply:
Good idea, updated